Backend/Remora.Discord.API/Json/Converters/Internal/VoicePayloadConverter.cs

```diff
@@ -0,0 +1,223 @@
+//
+//  VoicePayloadConverter.cs
+//
+//  Author:
+//       Jarl Gullberg <jarl.gullberg@gmail.com>
+//
+//  Copyright (c) 2017 Jarl Gullberg
+//
+//  This program is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU Lesser General Public License as published by
+//  the Free Software Foundation, either version 3 of the License, or
+//  (at your option) any later version.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU Lesser General Public License for more details.
+//
+//  You should have received a copy of the GNU Lesser General Public License
+//  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+//
+
+using System;
+using System.Linq;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using Remora.Discord.API.Abstractions.Voice.Gateway;
+using Remora.Discord.API.Abstractions.Voice.Gateway.Bidirectional;
+using Remora.Discord.API.Abstractions.Voice.Gateway.Commands;
+using Remora.Discord.API.Abstractions.Voice.Gateway.Events;
+using Remora.Discord.API.Voice.Gateway;
+using Remora.Discord.API.Voice.Gateway.Events;
+using Remora.Results;
+
+namespace Remora.Discord.API.Json
+{
+    /// <inheritdoc />
+    internal class VoicePayloadConverter : JsonConverter<IVoicePayload?>
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="VoicePayloadConverter"/> class.
+        /// </summary>
+        public VoicePayloadConverter()
```

Comment:
Could you add an `allowUnknownEvents` to this, similar as to how the normal gateway client is built? It's very useful for debugging.

Reply:
OK, I'll try it.

Reply:
Implemented in commit 530ca66.
Please check it.

Reply:
Looks pretty good :)

Reply:
Thanks!